docs/checkers/index.md

```diff
@@ -906,6 +906,24 @@ for student in students_info:
         cs_student_dict[student[0]] = student[1]
 ```
 
+(W3301)=
+
+### Nested min-max (W3301)
+
+This error occurs when there are nested calls of min or max instead of using a single min/max call.
```

Comment:
Use backticks around min and max to make them appear as `code font`, because they are the names of functions.

It would be good to provide more explanation here to explain why this is okay (you can pass in an arbitrary number of arguments to the functions).